build/Tests/Benchmark/Framework.Benchmark.vcxproj

```diff
@@ -221,7 +221,9 @@
   </ItemGroup>
   <ItemGroup>
     <ClangCompile Include="$(StarboardBasePath)\tests\Benchmark\BenchmarkSampleTests.mm" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\Benchmark\KVOBenchmarkTests.mm" />
```

Comment:
The multiple observer scenario is important.  We do see some improvement there so thats good to see.

---
In reply to: [108014282](https://github.com/Microsoft/WinObjC/pull/2323#discussion_r108014282) [](ancestors = 108014282)